e2etest/zt_managed_disks_test.go

```diff
@@ -59,10 +59,10 @@ func TestManagedDisks_NoOAuthRequired(t *testing.T) {
 }
 
 func TestManagedDisks_Snapshot(t *testing.T) {
-	if runtime.GOOS != "linux" {
-		t.Skip("Limit runs to Linux so no simultaneous runs occur")
-		return
-	}
+	// if runtime.GOOS != "linux" {
+	t.Skip("Limit runs to Linux so no simultaneous runs occur")
```

Comment:
Let's adjust the skip reason, or, better, put this behind a run flag. We may still want to run these tests locally, even if they don't work!

Reply:
Let me add the flag, thats the best option here.